src/current/v23.2/physical-cluster-replication-overview.md

```diff
@@ -122,11 +122,15 @@ Statement | Action
 
 ### Cluster versions and upgrades
```

Comment:
Ordering wise, i wonder if this makes sense to be at the bottom of the "get started" section

Reply:
Moved the section!